src/cloud/live/sens-data-over.md

```diff
@@ -6,202 +6,132 @@ functional_areas:
+1. Transfer the `/app/etc/config.php` file to your local project folder using `rsync` or `scp`.
 
    ```bash
-   rsync <SSH URL>:app/etc/config.php ./app/etc/config.php
+   rsync <SSH-URL>:app/etc/config.php ./app/etc/config.php
    ```
 
-1. Add and push the `config.php` file to the Git master branch.
+1. Add, commit, and push code changes to update a remote environment.
 
    ```bash
-   git add app/etc/config.php && git commit -m "Add system-specific configuration" && git push origin master
+   git add app/etc/config.php
    ```
 
-When you add the `config.php` file to Git, all build and deploy processes move static content deployment (SCD) to the _build_ phase. The method for the deployment uses the scope. The default option is [`quick`]({{ site.baseurl }}/guides/v2.3/config-guide/cli/config-cli-subcommands-static-deploy-strategies.html#static-file-quick). You can change the strategy by setting an environment variable for [`SCD_STRATEGY`]({{ site.baseurl }}/cloud/env/variables-deploy.html).
-
- {:.bs-callout-info}
-Once you add this file to your code, you should not delete it. If you need to remove or edit settings, you must manually edit the file to make changes.
-
-### Step 3 & 4: Push Git branch to Staging and Production {#push-git}
-
-Log in to the Magento Admin panel in those environments to verify the settings. Only configured settings display when you use the `scd-dump` command. Continue configuring the environment, if needed.
+   ```bash
+   git commit -m "Add system-specific configuration"
+   ```
 
-For Starter, when you push, the updated code pushes to the active environment. Merge the branch to Staging and Production. Complete any additional configurations in Staging and Production as needed.
+   ```bash
+   git push origin <branch-name>
+   ```
 
-For Pro, when you push to the Git branch, the Integration `master` environment updates. Deploy this branch to Staging and Production. Complete any additional configurations in Staging and Production as needed.
+After the deployment is complete, log in to the Magento Admin panel for the updated environment to verify the settings. Only configured settings display when you use the dump command. Continue to merge any additional configurations to the Staging and Production environments, as needed.
 
 ## Update configurations
 
-If you need to modify or remove any existing configuration settings in the `config.php` file, you will need to modify the file manually with a text editor. After completing edits or removals, you can push it to Git to update.
-
-To add new configurations, modify your environment through the Magento Admin panel and run the command again to generate the file. Any new configurations are appended to the code in the file. Push it to Git to update.
+To add new configurations, modify your environment through the Magento Admin panel and run the command again to generate the file. Any new configurations are appended to the code in the file.
```

Comment:
To add new configuration settings..   New settings are appended to the code in the `config.php` file